zcash_client_sqlite/src/scan.rs

```diff
@@ -448,7 +479,7 @@ pub fn decrypt_and_store_transaction<P: AsRef<Path>>(
         let value = output.note.value as i64;
 
         if output.outgoing {
-            let to_str = RecipientAddress::from(output.to).to_string();
+            let to_str = RecipientAddress::from(output.to).to_string(params);
```

Comment:
Hmm... seeing this in-situ, I dislike that we are using the same method name as the `ToString` standard trait, but with arguments. This should probably instead be `RecipientAddress::encode` or `RecipientAddress::encode_to_string`.